services/gossip/topic_benchmark_consensus.go

```diff
@@ -0,0 +1,98 @@
+package gossip
```

Comment:
nice! this is a very good feature of Golang you're making use of here, the ability to split a "class" to multiple files by context

Reply:
good feedback from Shai! achievement unlocked!

Reply:
:)